acc_test/resource_kafka_test.go

```diff
@@ -41,16 +44,26 @@ func TestKafkaResources(t *testing.T) {
 	configBytesAcl2, _ := ioutil.ReadFile("data/kafka_acl_create_acl_duplicate.tf")
 	configBytesAcl3, _ := ioutil.ReadFile("data/kafka_acl_list.tf")
 
+	// config bytes for Kafka Connect
+	kcConfigBytes, _ := ioutil.ReadFile("data/kafka_connect_create_cluster.tf")
+
 	kafkaUsername1 := "charlie1"
 	kafkaUsername2 := "charlie2"
 	kafkaUsername3 := "charlie3"
 	oldPassword := "charlie123!"
 	newPassword := "charlie123standard!"
 	kafkaNodeSize := "KFK-DEV-t4g.medium-80"
 	zookeeperNodeSize := "KDZ-DEV-t4g.small-30"
-	kafkaVersion := "2.7.1"
+	kafkaVersion := "apache-kafka:3.0.0.ic1"
+	kcVersion := "kafka-connect:3.0.0.ic1"
```

Comment:
Use 3.0.0 please, not the internal bundle name

Reply:
OK, just realized there's a new function to suppress the diff now